### DISCUSSION TITLE
5 x Red LED flash

### DISCUSSION THREAD

**Discussion by roses2uk** [ZuluSCSI/ZuluSCSI-firmware#68] [General]
So how do I get my money back from Rabbit Hole as no-one is answering Emails??

Brand new ZuluSCSI mini just arrived, but am very disappointed

Using a brand new FAT32 formatted micro SD card. Kingston Class 10 32GB

Copied 6 Roland format CD-ROM images to it, renamed them CD1.iso, CD2.iso, etc. Eject safely, insert card into ZuluSCSI.

Roland S760 is powered down, attach ZuluSCSI to rear SCSi port, power up.

ZuluSCSI powers on from buss, red LED flashes 5 times (= card missing), S760 does a SCSI buss check, finds no devices. Reinsert card, no difference.

Tried with another card Sandisk Class 10 16GB , same thing.

Advice please.

Also: Which firmware update file is needed for mini? Wiki says look for whether board is marked v1.0 or v1.1, but the plastic case of the mini is blurred and no PCB markings can be seen.

**Comment by aperezbios** [comment]
@roses2uk We'll get it figured out. No need to be disappointed just yet... Did you format the microSD card yourself, or are you using it as it came from the manufacturer? If you formatted it yourself, please share with us how you formatted it (which utility used, OS, etc) In order for the ZuluSCSI firmware to be able to detect SD cards, they _MUST_ be formatted with DOS/MBR style partition table. GPT/GUID-partitioned SD cards will not be detected, and will result in five LED flashes.

The latest firmware for V1.0 (Mini) hardware is here: https://github.com/ZuluSCSI/ZuluSCSI-firmware/releases/download/v1.0.8/ZuluSCSIv1_0-1.0.8-064ccdf.bin


On the release notes section, it does state clearly that "The ZuluSCSI Mini (DB25 version) is revision V1.0 hardware." See https://github.com/ZuluSCSI/ZuluSCSI-firmware/releases/tag/v1.0.8


If you have any other microSD cards around, it would be worthwhile to see if any other cards are detected. Preferably cards 8GB or larger.

**Reply by roses2uk** [reply]
Formatted in Win 7 myself as FAT32. Two brand new cards used, both above 8GB, do not have any others or money to buy them.

**Reply by PetteriAimonen** [reply]
With firmware version 1.0.5 and newer, GPT/GUID partition table results in one blink and the fallback raw SD card mapping.
But with older versions it gives 5 blinks.

To check the partition table type in Windows, you can use the "Disk Management" tool that comes with Windows.
Select the SD card and right click, select "Properties" and there it will say "Partition style" MBR (ok for ZuluSCSI) or GPT (needs to be repartitioned).
![](https://www.howtogeek.com/wp-content/uploads/2016/03/img_56e6297886fee.png?trim=1,1&bg-color=000&pad=1,1)

**Reply by roses2uk** [reply]
Why would my mini have older than v1.05 firmware? It's brand new, sent from Rabbit Hole 2 weeks ago. Nevertheless, I will try updating the firmware.

**Reply by roses2uk** [reply]
Yes, it says MBR, but I would expect that as it was formatted with the standard Windows formatting tool

**Reply by PetteriAimonen** [reply]
Ok, in that case it is not the partition table that is the problem. I wonder if there might have been some shipping damage to microSD card connector or similar. Alex can probably confirm what firmware version hardware currently ships with.

Because it is not detecting the SD card, you cannot upgrade firmware through that. Upgrading through USB with the [GD32 tool](https://www.gd32mcu.com/en/download?kw=dfu&lan=en) is possible. It requires opening the case to press down the bootloader button while connecting USB.

**Reply by roses2uk** [reply]
Nope, firmware upgrade from bin file does not work. Just 5 blinks all the time. The card slot seems indeed to be damaged. Not prepared to open the case which would void my guarantee. I'll contact Rabbit Hole.

**Reply by roses2uk** [reply]
With Alex's help, downloaded the GD32 tool and drivers, entered bootloader mode, successfully flashed with v1.0 firmware, but problem of 5 flashes persists even after inserting a valid card. This leads to the conclusion there is some kind of manufacturing fault. Item was very well and securely packed so transport damage can be ruled out. Hope I can get a replacement soon having had to wait 3 weeks for this one.

**Reply by roses2uk** [reply]
Anyone know where Alex Perez is? Or have another Email address for someone else at Rabbit Hole??

**Reply by sagamusix** [reply]
I'd hope you have already found a solution to your problem, but in case you haven't, or someone else comes across the same problem: Apparently the 5 flashes may also appear if there's insufficient current being delivered by the host. While on my Korg Triton this issue manifested as [infinite blinking](https://github.com/ZuluSCSI/ZuluSCSI-firmware/discussions/100), I have now encountered a Kurzweil K2500 where the diode on the termination power line blew, and for some reason in that case I got the 5 flashes, even though the same SD card worked when using the ZuluSCSI with a different device. Did you connect your ZuluSCSI to your PC with a USB cable while being plugged into your S760? If it works that way, the termination power on your device needs to be fixed (probably also a blown fuse or diode). If it still doesn't work with the extra USB power, it's probably really broken.

**Reply by aperezbios** [reply]
>  This leads to the conclusion there is some kind of manufacturing fault. Item was very well and securely packed so transport damage can be ruled out. Hope I can get a replacement soon having had to wait 3 weeks for this one.

I'm not dead yet, @roses2uk, but I have no way of correllating who you are here vs who you are via e-mail. I've responed to plenty of customer e-mails since September 28th. Your conclusion that there's likely to be a hardware fault is premature. There are a handful of other reasons why you might get five blinks, especially if you're using an old (or improperly-formatted SD card), which the silicon may not support.  Please send an e-mail, as again, I have no way of knowing what your name or e-mail address is, and therefore have no way of connecting the dots between here and e-mail.

**Reply by tcj** [reply]
Sorry to wake up an old issue, but my experience with SCSI2SD and ZuluSCSI Mini on the S-760 is that it simply must be powered externally.  Whether this means plugging it into a USB power source, or whether it means adding an additional SCSI device to the bus which supplies the necessary power, is up to you.  I have had success with both methods.
